sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Stick with present tense (we observe instead of we'll observe) - make sure that this applies to all of your work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>